upcloud/request/server_group.go

```diff
@@ -55,11 +55,21 @@ func (s GetServerGroupRequest) RequestURL() string {
 	return fmt.Sprintf("%s/%s", serverGroupBasePath, s.UUID)
 }
 
+// serverGroupPayload represents the data that we send to server group API.
+// It exists to allow some type transformations for improved user experience.
+type serverGroupPayload struct {
+	Labels       *upcloud.LabelSlice      `json:"labels,omitempty"`
+	Members      *upcloud.ServerUUIDSlice `json:"servers,omitempty"`
+	AntiAffinity string                   `json:"anti_affinity,omitempty"`
```

Comment:
The main reason I added this type is because API expects string (`"yes"` or `"no"`) in payload, while I think from the user perspective it's much better to use the `upcloud.Boolean` type

Reply:
I think that we don't need that separate payload type because `upcloud.Boolean`'s `MarshalJSON` already does that conversion?